API Server: Panic when fail to load webhook config file

**What happened**:
When parsing an audit webhook config file that errors (ca file does not exist), the apiserver will panic.

**What you expected to happen**:
Error gracefully with explanation to the error (ca authority does not exist).

**How to reproduce it (as minimally and precisely as possible)**:
```
$ etcd &
$ cat <<EOF > audit.yaml
apiVersion: audit.k8s.io/v1
kind: Policy
rules:
- level: RequestResponse
EOF
$ cat <<EOF > webhook-config.yaml
kind: Config
clusters:
- cluster:
    server: https://1.2.3.4:6443
    certificate-authority: ca.pem
  name: logstash
contexts:
- context:
    cluster: logstash
    user: ""
  name: default-context
current-context: default-context
preferences: {}
users: []
EOF
$ rm -f ca.pem
$ kube-apiserver --audit-webhook-config-file webhook-config.yaml --etcd-servers http://127.0.0.1:2379 --audit-policy-file audit.yaml 
W0305 22:18:40.033459    9933 services.go:37] No CIDR for service cluster IPs specified. Default value which was 10.0.0.0/24 is deprecated and will be removed in future releases. Please specify it using --service-cluster-ip-range on kube-apiserver.
I0305 22:18:40.033557    9933 server.go:656] external host was not specified, using 10.132.0.3
W0305 22:18:40.033569    9933 authentication.go:469] AnonymousAuth is not allowed with the AlwaysAllow authorizer. Resetting AnonymousAuth to false. You should use a different authorizer
I0305 22:18:40.034061    9933 server.go:153] Version: v1.19.0-alpha.0.597+8508875e4d400c
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x18425c5]

goroutine 1 [running]:
k8s.io/kubernetes/vendor/k8s.io/apiserver/plugin/pkg/audit/webhook.loadWebhook(0x7fff804bcd91, 0xa, 0x4544946, 0xc, 0x4544953, 0x2, 0x2540be400, 0x0, 0x0, 0x0, ...)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go:67 +0x195
k8s.io/kubernetes/vendor/k8s.io/apiserver/plugin/pkg/audit/webhook.NewBackend(0x7fff804bcd91, 0xa, 0x4544946, 0xc, 0x4544953, 0x2, 0x2540be400, 0x0, 0x0, 0x0, ...)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go:91 +0x87
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/options.(*AuditWebhookOptions).newUntruncatedBackend(0xc000229c30, 0x0, 0xc0003426e0, 0x0, 0x0, 0xc000325440)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/options/audit.go:605 +0x72
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/options.(*AuditOptions).ApplyTo(0xc000229b80, 0xc0005526c0, 0xc00061ee00, 0x51c25a0, 0xc000988c80, 0xc000325440, 0xc000d0cf70, 0xc000325360, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/options/audit.go:335 +0xa6a
k8s.io/kubernetes/cmd/kube-apiserver/app.buildGenericConfig(0xc00094adc0, 0xc0000e8000, 0x407ac40, 0x30, 0x30, 0xc00043dc00, 0xd0, 0xc000514410, 0x30, 0x30, ...)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:525 +0xf2a
k8s.io/kubernetes/cmd/kube-apiserver/app.CreateKubeAPIServerConfig(0xc00094adc0, 0x0, 0x0, 0xc0000e8000, 0x0, 0x0, 0xc000a534f8, 0x42ef01, 0x46d14e8, 0xc000a53508, ...)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:284 +0x50
k8s.io/kubernetes/cmd/kube-apiserver/app.CreateServerChain(0xc00094adc0, 0xc00029ccc0, 0x453b867, 0xc, 0xc000a53c48)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:175 +0x74
k8s.io/kubernetes/cmd/kube-apiserver/app.Run(0xc00094adc0, 0xc00029ccc0, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:155 +0x101
k8s.io/kubernetes/cmd/kube-apiserver/app.NewAPIServerCommand.func1(0xc00032f900, 0xc00008c720, 0x0, 0x6, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go:122 +0x104
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).execute(0xc00032f900, 0xc0000bc010, 0x6, 0x6, 0xc00032f900, 0xc0000bc010)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:826 +0x460
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc00032f900, 0x15f9886704418f88, 0x76c41e0, 0xc00006c750)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
k8s.io/kubernetes/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/spf13/cobra/command.go:864
main.main()
	_output/dockerized/go/src/k8s.io/kubernetes/cmd/kube-apiserver/apiserver.go:43 +0xcd
```

**Anything else we need to know?**:
[Here](https://github.com/kubernetes/kubernetes/blob/8508875e4d400c1454561aa6d879860984bcdf59/staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go#L67) we are assigning a null pointer when `err != nil`. We just need to add a standard `if err != nil { return nil, err }` before. Will follow up with a PR shortly.

**Environment**:
- Kubernetes version (use `kubectl version`):`kube-apiserver --version
Kubernetes v1.19.0-alpha.0.597+8508875e4d400c`

/kind bug
/area apiserver

--- a/staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go
+++ b/staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go
@@ -64,8 +64,12 @@ func retryOnError(err error) bool {
 func loadWebhook(configFile string, groupVersion schema.GroupVersion, initialBackoff time.Duration, customDial utilnet.DialFunc) (*webhook.GenericWebhook, error) {
 	w, err := webhook.NewGenericWebhook(audit.Scheme, audit.Codecs, configFile,
 		[]schema.GroupVersion{groupVersion}, initialBackoff, customDial)
+	if err != nil {
+		return nil, err
+	}
+
 	w.ShouldRetry = retryOnError
-	return w, err
+	return w, nil
 }
 
 type backend struct {

